common/network/iota_packet.c

```diff
@@ -22,3 +22,19 @@ void iota_packet_build(iota_packet_t* const packet, size_t const length,
   packet->source.port = port;
   packet->source.protocol = protocol;
 }
+
+void iota_packet_set_transaction(iota_packet_t* const packet,
+                                 iota_transaction_t const tx) {
+  flex_trit_t tx_flex_trits[FLEX_TRIT_SIZE_8019];
+  trit_t tx_trits[NUM_TRITS_SERIALIZED_TRANSACTION];
+
+  if (packet == NULL || tx == NULL) {
+    return;
+  }
+
+  transaction_serialize_on_flex_trits(tx, tx_flex_trits);
+  flex_trit_array_to_int8(tx_trits, NUM_TRITS_SERIALIZED_TRANSACTION,
```

Comment:
:thinking: we need an efficient `flex_trit_array_to_bytes` method I think

Reply:
@thibault-martinez could you add a tracker  '`// Tracked via #165` here so we don't forget about it?

Reply:
Done